### DISCUSSION TITLE
Add support for URL query parameters when posting new discussions or responses

### DISCUSSION THREAD

**Discussion by b0o** [community/community#3799] [Discussions]
A [similar feature](https://docs.github.com/en/github/managing-your-work-on-github/about-automation-for-issues-and-pull-requests-with-query-parameters) already exists for issues and PRs. It would be nice if this idea was extended to discussions.

For example,  to compose a new discussion in this repo under the "Discussions Feedback" category, with the title pre-filled as "Hello world" and the body text "This is a test":

`https://github.com/github/feedback/discussions/new?category=Discussions-Feedback&title=Hello%20world&body=This%20is%20a%20test`

![2021-04-16_19-58-45_region](https://user-images.githubusercontent.com/21299126/115099978-6216d280-9eee-11eb-8af2-3c3bd07aca4b.png)

---

In addition to supporting query parameters when creating a new top-level discussion, it would be nice if it was possible to create links to reply to a specific discussion and/or comment with the body pre-filled. 

For example, to compose a reply to this discussion with the body text pre-filled as "Hello world":

`https://github.com/github/feedback/discussions/3799?reply-to=discussion-3325328&body=Hello%20world`

![2021-04-16_19-54-16_region](https://user-images.githubusercontent.com/21299126/115099871-a2298580-9eed-11eb-964a-56ea468300bd.png)

My use case is described a bit more here: https://github.com/github/feedback/discussions/2937#discussioncomment-623022, but I can see this being useful for other purposes as well.

**Comment by bdeo** [comment]
We've been looking for this feature for a while within the [Vuetify](https://github.com/vuetifyjs) org, specifically for posting new discussions. Is this a planned feature?
